test/common/cdp.py

```diff
@@ -167,7 +178,7 @@ def browser_cmd(self, cdp_port, env):
             raise SystemError(self.browser + " is not installed")
 
         if self.browser == "chromium":
-            return [exe, "--headless", "--disable-gpu", "--no-sandbox", "--disable-setuid-sandbox",
+            return [exe, "--headless" if self.show_browser is False else "", "--disable-gpu", "--no-sandbox", "--disable-setuid-sandbox",
```

Comment:
That looks a bit hackish -- chromium is cool with empty arguments?

Reply:
Tested it and seems fine